src/components/mdx/email-subscribe-widget.tsx

```diff
@@ -0,0 +1,201 @@
+                        <span className="pl-2">TypeScript</span>
+                      </label> */}
+                      <button
+                        className={`bg-blue-600 text-white rounded-md font-semibold p-1 ${
+                          errors.email &&
+                          touched.email &&
+                          'opacity-50 cursor-not-allowed'
+                        } ${isSubmitting && 'opacity-50 cursor-not-allowed'}}`}
+                        type="submit"
+                        disabled={
+                          (errors.email && touched.email) ||
+                          !values.email ||
+                          isSubmitting
+                        }
+                      >
+                        {isSubmitting ? 'SUBMITTING...' : 'SIGN UP'}
+                      </button>
+                    </Form>
+                  </div>
+                )
+              } else {
+                return (
+                  <div>
+                    <h2 className="text-3xl leading-tight font-bold">
+                      You're signed up!
+                    </h2>
+                    <p className="py-2">Check your email for a confirmation</p>
+                  </div>
+                )
+              }
+            }}
+          </Formik>
+        </div>
+
+        <div className="hidden sm:flex sm:flex-col p-6 rounded-r-md text-gray-800 bg-white dark:bg-gray-800 dark:text-white border-y-2 border-r-2 border-gray-300 dark:border-none">
+          <div className="flex flex-row items-center pb-4 dark:text-white">
+            <Image
+              src="https://res.cloudinary.com/dg3gyk0gu/image/upload/v1569690305/transcript-images/eggo_new.png"
+              alt=""
+              title=""
+              width={58}
+              height={58}
+            />
+            <span className="pl-1 dark:pl-2 text-3xl font-semibold">
+              egghead.io
+            </span>
+          </div>
+
+          <h2 className="text-lg font-semibold leading-snug">
+            We're here to help.
+          </h2>
+          {/* <p className="pt-2">
+            Sign up for one or more of our <b>FREE</b> email courses
+          </p> */}
+          <p className="pt-2">
+            Sign up for our <b>FREE</b> email course
+          </p>
+          <ul className="pt-4">
+            <li className="pb-2">
+              ⭐️ <b>Portfolio Building:</b> Learn how to build a badass
+              developer portfolio so you can land that next job
+            </li>
+            {/* TODO: Make these campaigns in CIO */}
+            {/* <li className="pb-2">
+              ⭐️ <b>Full-Stack in 2023:</b> Build several full-stack apps using
+              the latest tools and trends (Open AI, Next.js, Remix, and more)
+            </li>
+            <li>
+              ⭐️ <b>TypeScript:</b> Build a solid foundation in TypeScript and
+              learn some advanced techniques along the way
+            </li> */}
```

Comment:
The initial plan was to create these campaigns but in the interest of saving time I'm starting with just the portfolio campaign since it already exists. 

Phase two of this will be to make this CTA dynamic using data from Sanity.